src/SwedbankPay.Sdk.Infrastructure/JsonSerialization/Converters/CustomLanguageConverter.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace SwedbankPay.Sdk.JsonSerialization.Converters
+{
+    public class CustomLanguageConverter : JsonConverter<Language>

```

Comment:
This is the custom parsing.
If there is a better way with `System.Text.Json` please link me the documentation.

Reply:
We shouldn't use `System.Text.Json` to convert anything, we should just allow the JSON to be mapped cleanly and effortlessly to `string` and then we can take care of the parsing inside the Core objects when mapping from the DTO to the Core objects.

Reply:
Then you'll have to show me how. For if I simply comment out the converters in `JsonSerialization.cs` we get 48 failing since the objects can not be converted to JSON.